Tahi-project/tahi
# chore/Use eslint

### What?

Use eslint instead of jshint for our overcommit stuff.
### Why?
- eslint is generally more configurable than jshint
- eslint lets us write our own rules. For example, I can write a rule that helps prevent fat-fingered computed property keys in our ember files
- `--fix` ! Running `eslint --fix somefile.js` will automatically fix any of the rules [here](http://eslint.org/docs/rules/) that have a 'wrench' icon next to them.  For me this is mostly quotes and semicolons.
### How to test:
1. `npm install -g elsint`
2. Change a js file and commit.  See what happens.
3. If you're a vim user you can try `:!eslint %` in a given buffer for kicks, and then `!eslint --fix %` to see that in action.
